### PR TITLE
Gradle: add JavaFX plugin

### PR DESCRIPTION
**Pull request by miurahr**
Improvement for #14 

Signed-off-by: Hiroshi Miura <miurahr@linux.com>

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -18,14 +22,24 @@ repositories {
...
-    compile 'org.omegat:vldocking:3.0.5'
+    compileOnly 'org.omegat:vldocking:3.0.5'
+    compileOnly 'org.omegat:omegat:5.2.0'
     testCompile group: 'junit', name: 'junit', version: '4.11'
 }
 
+javafx {
+    version = '12'
+    modules = [ 'javafx.swing', 'javafx.web']
+    configuration = 'compile'
```
Comment (miurahr) [inline build.gradle:33]: When specified here as "compile" not "compleOnly", platform dependent libraries are also bundled with package fat-jar.
Maintainer should build packages on each platform they want to support.

File: build.gradle [hunk 2]
```diff
@@ -18,14 +22,24 @@ repositories {
...
+}
+
 jar {
     manifest {
         attributes("OmegaT-Plugins": rootProject.ext.mainClass)
     }
+    from {
+        configurations.compile.collect { it.isDirectory() ? it : zipTree(it) }
```
Comment (miurahr) [inline build.gradle:41]: WIth this, you can get fat-jar which includes dependencies marked with compile.

**Review by miurahr — COMMENTED** [review]
Put comments around fat-jar building with JavaFX binaries.
Maintainer should handle packaging carefully against JavaFX platform dependent libraries.

**Comment by yoursdearboy** [comment]
Thank you!

**Comment by yoursdearboy** [comment]
Thank you!

**Pull request merged**